.github/workflows/build_docker.yml

```diff
@@ -12,6 +12,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: docker/setup-buildx-action@v2
-      - uses: docker/build-push-action@v3
+      - uses: docker/build-push-action@v4
         with:
-          tags: segformer:latest
+          context: .
+          tags: segformer_playground:latest
+          load: true
+          cache-to: type=inline
+
+      - name: Run pytest
+        run: docker run -v ${{ github.workspace }}:/home/segformer_playground segformer_playground pytest
```

Comment:
I'm getting an error here but I suspect it's caused by running out of disk space earlier, see https://stackoverflow.com/questions/75536771/github-runner-out-of-disk-space-after-building-docker-image